client/a8c-for-agencies/components/a4a-migration-offer-v2/index.tsx

```diff
@@ -25,7 +25,11 @@ const MigrationOfferV2 = () => {
 
 			<div className="a4a-migration-offer-v2__main">
 				<h3 className="a4a-migration-offer-v2__title">
-					{ translate( 'Special limited-time migration offer for our partners' ) }
+					{ translate( 'Special limited time migration offer' ) }
+
+					<Button className="a4a-migration-offer-v2__view-toggle-mobile" onClick={ onToggleView }>
```

Comment:
We need an additional button dedicated to mobile view, which will only appear in narrow screens.